core/imt/inc/ROOT/TExecutor.hxx

```diff
@@ -188,18 +188,26 @@ private:
    auto Map(F func, std::initializer_list<T> args, R redfunc, unsigned nChunks) -> std::vector<typename std::result_of<F(T)>::type>;
 
    ROOT::EExecutionPolicy fExecPolicy;
+
+// For code simplification. Fallbacks to TSequentialExecutor (#else cases) are never instantiated.
```

Comment:
what do you mean with "are never instantiated"?

Reply:
That the unique_ptrs remain empty, they never own an instance of TSequentialExecutor in those particular cases. The wording is incorrect.

Reply:
ok so maybe something like:

```suggestion
// When they are not available, we use a placeholder type instead of TThreadExecutor or TProcessExecutor. The corresponding data members will not be used.
```